src/DotLiquid/Hash.cs

```diff
@@ -123,15 +104,20 @@ private static void FromAnonymousObject(object anonymousObject, Hash hash, bool
             return expr.Compile();
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DotLiquid.Hash">Hash</see> class and populates it with the contents of a dictionary
+        /// </summary>
+        /// <param name="dictionary">The Dictionary object</param>
+        /// <returns></returns>

```

Comment:
Nit: avoid empty XML elements in comments. Each element can be omitted entirely if there is nothing to say. Or add what you see as appropriate in the returns element.